Backpack-SwiftUI/Dialog/Classes/BPKDialogView.swift

```diff
@@ -0,0 +1,171 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+/// This extension provides a set of convenience methods to display a dialog that works as a view.
+/// It's usage is discouraged in favour of the `BPKDialog` view. Usage of this view should be limited to cases where
+/// presenting a new view controller is not possible. (`BPKDialog` uses a view controller to present the dialog
+/// behind the scenes).
+public extension View {
+    /// Displays a success dialog with a title, text and a list of buttons.
+    func bpkSuccessDialogView(
```

Comment:
@frugoman As I understand it, making the accessibility announcement that the dialog has been presented is now the responsibility of the consumer since we are not using `UIViewController.present`. Would it be possible to require a localised string in each of these interfaces that is the voice-over announcement to use when presented, and then `DialogContainerViewModifier` would have the responsibility to make the announcement. This ensures anyone using the interface has correct voice-over call outs. WDYT?

Reply:
@gert-janvercauteren what do you reckon would be the best approach? the dialog component does not know what the content of the dialog is, so having the event in the component seems weird.
(what the screen reader needs to say when a dialog is presented is not a specific string, but rather focus on one given part of the screen, and read that.)

I'm not sure forcing an interface that requires a string or similar is the way to go here